resources/snap_packs/parameters/lob_version.yml

```diff
@@ -0,0 +1,16 @@
+lob-version:
+  in: header
+
+  name: Lob-Version
+
+  required: true
```

Comment:
Set this header to required. Snap packs cannot be created if the header isn't set.